src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct

```diff
@@ -69,6 +73,10 @@
     <GuidSymbol name="guidWorkspaceExplorerToolWindowPackageCmdSet" value="{cfb400f1-5c60-4f3c-856e-180d28def0b7}">
       <IDSymbol name="idmWSE_ContextMenu" value="0x0002"/>
     </GuidSymbol>
+
+    <GuidSymbol name="guidManagedProjectSystemClientMainMenuCmdSet" value="{d309f791-903f-11d0-9efc-00a0c911004f}">
+      <IDSymbol name="groupidCtxtMenuEditProjectFile" value="0x0237"/>
```

Comment:
Can you add a comment explaining where else these two values are defined?

Reply:
done

Reply:
Next time, please use the names as they are defined within the shell, for example, IDG_VS_CTXT_PROJECT_EDITFILE.

Reply:
I wanted to use the original name, but it was going to look different from the rest of the file.